bites.py

```diff
@@ -138,6 +140,7 @@ def print_scores(self):
     Prints each player's name and shows how many points they have.
```

Comment:
The function description should also indicate that the scores are calculated before they are printed. 
Optionally, consider renaming the function to `calculated_and_print_scores()`